submissions/EmmanuelIdiaghe/php/admin.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+//session_start();
+//$user_identity = $_SESSION['sess_id'];
+
+#SetUp Connection
+$usernameDB = "dufuna";//database username
+$passwordDB = "dufuna12345";//database password
+
+function debug_to_console($data) {
+    $output = $data;
+    if (is_array($output))
+        $output = implode(',', $output);
+
+    echo "<script>console.log('Debug Objects: " . $output . "' );</script>";
+}
+
+try {
+    $pdo = new PDO("mysql:host=localhost", $usernameDB, $passwordDB);
+    $pdo->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
+    echo "Connected Successfully. Host info: " . $pdo->getAttribute(constant("PDO::ATTR_CONNECTION_STATUS")) . "<br>";
+
+} catch (PDOException $e) {
+    die ("ERROR: Could not connect. " . $e->getMessage()) . "<br>";
+}
+
+#Create Database
```

Comment:
You are expected to work with the already existing e-commerce database and tables you created in the previous task. You are not creating a new one

Reply:
Alright, done!